docs/pages/eas-update/debug-advanced.mdx

```diff
@@ -1,31 +1,49 @@
 ---
-title: Debug EAS Update
-sidebar_title: Debug
-description: Learn how to debug EAS Update.
+title: Advanced EAS Update Debugging
+sidebar_title: Advanced
+description: Learn advanced strategies on how to debug EAS Update.
 ---
 
 import ImageSpotlight from '~/components/plugins/ImageSpotlight';
 import { Terminal } from '~/ui/components/Snippet';
 
-It's important to tell the current state of our app at any given time. We built EAS Update with this in mind. Once we know which updates are running on which builds, we can make changes so that our apps are in the state we expect and desire. This guide sets out to show how we can verify our EAS Update and expo-updates configuration so that we can find the source of problems like an app not showing a published update.
+After verifying our EAS Update configuration in our [basic guide](/eas-update/debug), we can move on to more advanced debugging strategies. The following sections describe common classes of problems and strategies on how to tackle them.
 
-## expo-updates configuration
+## General strategies
+We recommend trying these strategies before using the more specific ones mentioned in this guide. 
```

Comment:
The "we" in this refers to Expo as a company versus the first paragraph referring to the developer as we. Thou shalt make this consistent.